cmd/action/internal/update/update_test.go

```diff
@@ -0,0 +1,81 @@
+package update
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestCheckForUpdate(t *testing.T) {
+	url := "https://github.com/ariga/atlas/releases/tag/test"
+	tests := []struct {
+		name           string
+		currentVersion string
+		latestVersion  string
+		store          *LatestRelease
```

Comment:
Unit tests should be  testing the public api, put the state file text instead of the results of its parsing

Reply:
changed that store is saved externally.